app/models/name_sorter.rb

```diff
@@ -254,40 +254,13 @@ def add_name(spl_line, timestamp = nil)
     return unless name_parse.has_synonym
 
     @has_new_synonyms = true
-    if name_parse.find_synonym_names.empty?
+    if name_parse.find_synonym_names(user).empty?
       @new_name_strs.push(name_parse.synonym_search_name)
     end
     # Keep names in addition to parse for the chosen filter
     @synonym_data.push([name_parse, names])
```

Comment:
It looks like the attribute reader `@synonym_data` is no longer needed in this class, if the method `create_new_synonyms` is going away (below). That's the only place it's used.

Reply:
Mmm, but you could also say ripping out the method is outside this PR. Isn't this is how confusing dead code starts? 

If a PR is ripping out the only method that uses the code, to me there's no doubt cleanup from that belongs in the same PR. But maybe you reckon `@synonym_data` might be useful again at some point.

Reply:
Turned out to be easy enough to get rid of it.